server/src/main/java/io/horizen/helpers/Helper.java

```diff
@@ -0,0 +1,155 @@
+package io.horizen.helpers;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.horizen.config.Settings;
+import io.horizen.data_types.VotingProposal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Base64;
+import java.util.Date;
+
+public class Helper {
+
+    private static final Logger log =  LoggerFactory.getLogger(Helper.class);
+
+    private static Settings settings;
+
+    public static void initialize(Settings mySettings) {
```

Comment:
Maybe we can add a check on settings == null for avoid double init?

Reply:
Done